text/000-2018-2019-roadmap.md

```diff
@@ -0,0 +1,88 @@
+
+Building on the simplicity of Glimmer components, we’ll also implement **template imports**, which make it explicit which helper or component is being invoked, and **named blocks**, which make it even easier to build ergonomic, reusable components.
+
+Lastly, we’ll **simplify and modernize Ember Data,** including:
+
+- **Easier on-ramp for new users** by eliminating the need for adapters and serializers for simple use cases based on `fetch()`.
+- **Increased velocity and stability** through improved, modularized internals.
+- **More flexibility** when overriding built-in behavior, by exposing low-level primitives as public API.
+
+### Modernize our build system
+
+Last year, we started work on Embroider, an overhaul of the Ember CLI compilation pipeline. This year, **we will put the finishing touches on Embroider** and start migrating the Ember ecosystem to this modernized build.
+
+By using standard JavaScript modules, Embroider integrates with popular packagers like [webpack](https://webpack.js.org/) and [r](https://rollupjs.org/guide/en/)[ollup](https://rollupjs.org/guide/en/), allowing us to **ship Ember itself as npm packages in the** `**@ember**` **namespace**. 
+
+These packagers also unlock build time optimizations like **tree-shaking,** and allow Embroider to produce smaller payloads with **route-based code-splitting.** Most importantly, they break down any remaining barriers between Ember and the rest of the JavaScript ecosystem.
+
+### Better a11y by default
+
+Ember applications should be accessible to everyone. Unfortunately, even seemingly small mistakes can make your app difficult or impossible to use with assistive technology like screen readers. We will do more to **improve the out-of-the-box accessibility of Ember applications**, and provide tools for helping applications stay accessible as they grow.
+
+- **Fix router accessibility** so that page navigations are correctly announced by screen readers, without needing a third-party addon.
+- Include **more a11y lint rules** to catch common accessibility mistakes.
+- **Document accessibility best practices** and treat accessibility as a first-class concern, as we do with automated testing.
+- Incorporate **accessibility checks** into the built-in test helpers.
+- **Engage with standards bodies** to help fill the gaps in existing web accessibility APIs.
+
+### Optimize for growth
+
+There are more people building web applications than ever, and Ember must adapt to their changing needs and expectations in order to stay relevant.
+
+This year, **we’ll share with the world how Ember Octane is modern, productive, and _fun_.** Through blog posts, videos, social media, meetups, and conferences, we will share our knowledge and experiences with the wider JavaScript community and encourage them to give Octane a try.
```

Comment:
This, to me, is one of the most important pieces of this roadmap, and I hope people do not gloss over it. I can't remember the last time I attended a conference/meetup that was not specifically focused on Ember and saw someone giving a talk about Ember. We need to make a concerted effort at outreach, and I hope that someone will be writing a follow-up RFC detailing exactly how we will accomplish this rather than simply jumping straight from this roadmap into "executing" at this broad-stroke level.